src/hardware/adlib_gold.cpp

```diff
@@ -345,6 +350,12 @@ void AdlibGold::Process(const int16_t *in, const uint32_t frames, float *out) no
 
 		frame = stereo_processor->Process(frame);
 
+		limiter_in[0] = frame.left;
+		limiter_in[1] = frame.right;
+		soft_limiter->Process(limiter_in, 1, limiter_out);
+
+//		out[0] = limiter_out[0];
+//		out[1] = limiter_out[1];
 		out[0] = frame.left;
 		out[1] = frame.right;
```

Comment:
The limiter can work on whole segments, and should (I think) allow it to shape the waveform a bit better (might be able to take in all of `out` as it's input)